services/QuillLMS/spec/models/concept_feedback_spec.rb

```diff
@@ -74,39 +75,116 @@
       it 'adds the translations to the data' do
         expect(subject["translatedDescription"]).to eq(translation)
       end
+
+      context 'there is only a gengo translation' do
+        let(:translated_text) { create(:gengo_translated_text, translation: translation)}
+
+        it 'adds the translations to the data' do
+          expect(subject["translatedDescription"]).to eq(translation)
+        end
+      end
+
+      context 'there is a gengo and open ai translation' do
+        let(:gengo_text) { create(:gengo_translated_text)}
+
+        before do
+          concept_feedback.english_texts.first.translated_texts = [gengo_text, translated_text]
+        end
+
+        it 'defaults to the open_ai translation' do
+          expect(subject["translatedDescription"]).to eq(translation)
+        end
+
+        context 'passed in gengo as an option' do
+          let(:options) { {source_api: TranslatedText::GENGO_SOURCE} }
+
+          it 'returns the gengo translation' do
+            expect(subject["translatedDescription"]).to eq(gengo_text.translation)
+          end
+        end
+
+      end
     end
   end
 
-  describe '#translation(locale:)' do
+  describe '#translation(locale:, source:)' do
     let(:locale) {"es-la"}
+    let(:translation_locale) { locale }
+    let(:source) { TranslatedText::OPEN_AI_SOURCE }
```

Comment:
If we call this `source_api` it lets us simplify creation calls below with `create(:translated_text, source_api:, ...)`